marge/bot.py

```diff
@@ -153,15 +153,21 @@ def _process_merge_requests(self, repo_manager, project, merge_requests):
                 log.exception('BatchMergeJob failed: %s', err)
         log.info('Attempting to merge the oldest MR...')
         merge_request = merge_requests[0]
-        merge_job = single_merge_job.SingleMergeJob(
+        merge_job = self._get_job(
+            project=project, merge_request=merge_request, repo=repo,
+            options=self._config.merge_opts,
+        )
+        merge_job.execute()
+
+    def _get_job(self, project, merge_request, repo, options):
```

Comment:
There's also a `BatchMergeJob`, so perhaps this method could be clearer about what type of job it's referring to? E.g. `_get_single_merge_job`.

Reply:
maybe `_get_single_job` would fit ?